editoast/src/models/projects.rs

```diff
@@ -219,8 +219,8 @@ impl List<Ordering> for ProjectWithStudies {
     ) -> Result<PaginatedResponse<Self>> {
         let ordering = ordering.to_sql();
         sql_query(format!(
-            "SELECT t.*, COUNT(study.*) as studies_count FROM osrd_infra_project t
-            LEFT JOIN osrd_infra_study study ON study.project_id = t.id
+            "SELECT t.*, COUNT(study.*) as studies_count FROM project t
+            LEFT JOIN study study ON study.project_id = t.id
```

Comment:
```suggestion
            LEFT JOIN study ON study.project_id = t.id
```
Also, the alias `t` is a little bit confusing and is used in some other queries of the same type.